docs/wcurl.md

```diff
@@ -0,0 +1,124 @@
+---
+c: Copyright (C) Daniel Stenberg, <daniel.se>, et al.
+SPDX-License-Identifier: curl
+Title: wcurl
+Section: 1
+Source: wcurl
+See-also:
+  - curl (1)
+  - trurl (1)
+Added-in: 7.1
+---
+
+# NAME
+
+wcurl - download URLs
+
+# SYNOPSIS
+
+**wcurl [options / URLs]**
+
+# DESCRIPTION
+
+**wcurl** is a simple curl wrapper which lets you use curl to download files
+without having to remember any parameters.
+
+Simply call **wcurl** with a list of URLs you want to download and **wcurl**
+picks sane defaults.
+
+If you need anything more complex, you can provide any of curl's supported
+parameters via the **--curl-options** option. Just beware that you likely
+should be using curl directly if your use case is not covered.
+
+By default, **wcurl** does:
+
+## Encode whitespaces in URLs
+
+## Download multiple URLs in parallel
+if the installed curl's version is \>= 7.66.0
+
+## Follow redirects
+
+## Automatically choose a filename as output
+
+## Avoid overwriting files
+if the installed curl's version is \>= 7.83.0 (**--no-clobber**)
+
+## Perform retries
+
+## Set the downloaded file timestamp
+to the value provided by the server, if available
+
+## Default to https
+if the URL does not contain any scheme
+
+## Disable curl's URL globbing parser
```

Comment:
This is an outdated version of the manpage

Reply:
Ah yes, I need to get the markdown version into the wcurl repo first, then use that here...